app/screens/settings/general/settings.js

```diff
@@ -302,7 +302,8 @@ class Settings extends PureComponent {
                     />
                     }
                     <SettingsItem
-                        defaultMessage='About Mattermost'
+                        defaultMessage='About {appTitle}'
+                        messageValues={{appTitle: this.props.config.SiteName || 'Mattermost'}}
```

Comment:
can you find a way to make this a constant object to avoid unnecessary re-renders?

Reply:
@enahum why would this cause unnecessary re-renders? It'll re-render only on config changes and I assume those are pretty rare, no? Do you have a suggestion on improving this?

Reply:
@reflog ok lets see, config changes and the the Setting screen re-renders cause of that change in the props and it makes sense, BUT `SiteName` has not changed, just some other random property in the config object. What will happen here, which is not a huge deal is that `messageValues` will have a different object than the one that you passed the first time around causing an unnecessary re-render of the `SettingsItem` component as well.

is not a heavy component so I'm ok with this, but always consider the fact that passing a new object every time as a prop will cause re-renders ;)

Reply:
👍